pkg/util/selector/selector.go

```diff
@@ -0,0 +1,18 @@
+package selector
+
+import (
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/loft-sh/vcluster/config"
+)
+
+func StandardLabelSelectorMatches(obj client.Object, labelSelector config.StandardLabelSelector) bool {
```

Comment:
We could make this a function of config.StandardLabelSelector (`func (s *StandardLabelSelector) Matches(obj client.Object) bool`) and move it into config package instead of creating a new package just for this small function.

Reply:
Sure!